Iced 0.13

wash2 opened the pull request.

wiiznokes commented:
Looks like `Command` is not available anymore ?
![screenshot-2024-10-18-15-47-07](https://github.com/user-attachments/assets/1c8957b8-5327-4bbb-9c74-8d775feea22d)

mmstick commented:
@wiiznokes They're called Tasks now

wiiznokes commented:
Ah yeah, i forgot thanks

mmstick commented:
Noticing a lot of these logs whenever any redraw happens.

```
2024-10-18T15:57:50.175716Z  INFO wgpu_core::device::resource: Device::maintain: waiting for submission index 133
```

wiiznokes commented on src/widget/toggler.rs:
Could this file be removed, and the helper from Iced exported instead ? (to allow a disabled state)

mmstick replied:
Just needs to remove the requirement on supplying an on_toggle function.

wash2 commented:
> wgpu_core::device::resource: Device::maintain: waiting for submission index

This seems to be an issue with the 22.0 release of wgpu, that has since been reduced to a trace, and moved into the if statement. https://github.com/gfx-rs/wgpu/blob/278d278b28ff7b424dc7dde0c0d7519e8d839ec1/wgpu-core/src/device/resource.rs#L446
Fixed by: https://github.com/gfx-rs/wgpu/commit/841e7c885713983a354a814e5635bf43c1e37f56

Adam-Cosner commented:
cosmic::theme and cosmic::style need to be updated to reflect the new 0.13 style system
